docs/security/trust/content_trust.md

```diff
@@ -152,14 +152,36 @@ In an environment where `DOCKER_CONTENT_TRUST` is set, you can use the
 `--disable-content-trust` flag to run individual operations on tagged images
 without content trust on an as-needed basis.
 
-```bash
-$  docker pull --disable-content-trust docker/trusttest:untrusted
+Consider the following Dockerfile that uses an untrusted base image:
+
+```
+$  cat Dockerfile
+FROM docker/trusttest:latest
+RUN echo
+```
+
+In order to build a container successfully using this Dockerfile, one can do:
+
+```
+$  docker build --disable-content-trust -t <username>/nottrusttest:testing .
```

Comment:
Hm, would it make sense for this to be `<username>/trusttest:latest`?  Just because later, at the end, we want there to exist an unsigned `latest` image (due to [this comment](https://github.com/docker/docker/pull/23513/files#diff-9aa88e5a626d5840f1df9a54bf3a205dR277))

And then similarly, one line 177, `docker push --disable-content-trust <username>/trusttest:latest`

Sorry, this is kinda confusing.  :|  Do you think it would be helpful to also include a little chart up above summarizing the following:

- `docker/trusttest:latest` exists, and is unsigned
- We will create `<username>/trusttest:latest`, but not sign it
- We will create `<username>/trusttest:testing` and `<username>/otherimage:latest`, and sign both